modules/authorization/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable account_id {
+  description = "Account ID where the authorized aggregator exists"
+  type        = string
+}
+
+variable region {
+  description = "Region where the aggregator exists"
```

Comment:
Do you want to say "Region where the authorized aggregator exists"?

Reply:
Makes sense to be consistent. Fwiw, you can use the "suggest" option in the GitHub review to make it easier to accept changes...